17-react-native/classworks/Shevtsov_Vladimir/task/resources/comics.selector.js

```diff
@@ -0,0 +1,24 @@
+import { uniqBy } from 'lodash';
+
+export const getData = (state) => state.comics.data;
+export const getError = (state) => state.comics.error;
+export const getFetching = (state) => state.comics.fetching;
+
+export const getChars = (state) => {
+  const data = state.comics.data;
+  const results = data?.data?.data?.results ?? [];
+
+  const chars = uniqBy(results, "id").map((obj) => (
```

Comment:
why uniqBy? what is the situation when we have items with the same id in results?